controllers/helmchart_controller_test.go

```diff
@@ -697,14 +697,20 @@ var _ = Describe("HelmChartReconciler", func() {
 			When("Setting valid valuesFile attribute", func() {
 				updated := &sourcev1.HelmChart{}
 				Expect(k8sClient.Get(context.Background(), key, updated)).To(Succeed())
-				chart.Spec.ValuesFile = "./charts/helmchart/override.yaml"
+				updated.Spec.ValuesFile = "./testdata/charts/helmchart/override.yaml"
```

Comment:
I believe that @nniikkoollaaii is correct and this line needed to be corrected.

Reply:
https://github.com/fluxcd/source-controller/pull/305#discussion_r608419096

Reply:
Ya this was commited, this was just from my lazy print before the `wt.Commit` call to verify that the GitRepository's root was `testdata/charts/helmchartwithdeps`. 😁